### DISCUSSION TITLE
bookmarking or sticky indivisual posts within bluesky app.

### DISCUSSION THREAD

**Discussion by itaru2622** [bluesky-social/atproto#2627] [General Discussion]
for accounts, list is availabe to keep accounts like bookmarking, beside following them.
but no similar feature for posts within app.
then It needs to keep the post outside of app via sharing post to other app such as mailer.
it is nice to have a feature to keep individual posts within app  somehow.
then,  it is easy to "read this later".

**Comment by dolciss** [comment]
FYI, here is a similar discussion (no replies) #2000
and a pull request  #1771  that was tried but closed.
